.github/workflows/flowzone.yml

```diff
@@ -95,6 +95,16 @@ on:
         type: string
         required: false
         default: registry.npmjs.org
+      ignore_versionbot_branch:
+        description: "Ignores the branch versioned by Versionbot and will create a version from original source"
+        type: boolean
+        required: false
+        default: false
+      skip_versioning:
+        description: "Ignores the branch versioned by Versionbot and will create a version from original source"
```

Comment:
Let's remove the mention of versionbot here. We can clarify in the README that versionbot is always ignored and it will either use versionist or skip versioning.